### ISSUE TITLE
Lower button on the Thinkpad Pen Pro 2 not working (or not configured)

### ISSUE THREAD

**Issue by Photon89** [linuxwacom/xf86-input-wacom#171]
Hi guys, I just did a fresh install on my new laptop, a Thinkpad L13 Yoga. The stylus is a Thinkpad Pen Pro 2. It has two buttons on the side and no button on the cap. The stylus itself works, also one of the buttons works (tested in Xournal++). However, the other button does not work. 

Here is the list of devices being recognized:

```
$ xsetwacom list
Wacom Pen and multitouch sensor Pen stylus	id: 16	type: STYLUS    
Wacom Pen and multitouch sensor Pen eraser	id: 17	type: ERASER    
Wacom Pen and multitouch sensor Finger touch	id: 18	type: TOUCH     
```

At first I thought, id 16 is the stylus with no button pressed, while id 17 is the stylus with the working button pressed, so another entry for the second button is missing. However, I get exactly the same output with my old Thinkpad X200T which doesn't even have touch input. 

So maybe the buttons are hidden inside the stylus device id16. I get the following detailed output for this device:

```
$ xsetwacom get "Wacom Pen and multitouch sensor Pen stylus" all
Option "Area" "0 0 29376 16524"
Button: Actions are not supported by xorg.conf. Try shell format (-s) instead.
Button: Actions are not supported by xorg.conf. Try shell format (-s) instead.
Button: Actions are not supported by xorg.conf. Try shell format (-s) instead.
Button: Actions are not supported by xorg.conf. Try shell format (-s) instead.
Option "DebugLevel" "0"
Option "CommonDBG" "0"
Option "Suppress" "2"
Option "RawSample" "4"
Option "PressCurve" "0 0 100 100"
Option "Mode" "Absolute"
Option "TPCButton" "off"
Option "Touch" "off"
Property 'Wacom Hardware Touch Switch' does not exist on device.
Option "Gesture" "off"
Option "ZoomDistance" "0"
Option "ScrollDistance" "0"
Option "TapTime" "250"
Option "CursorProx" "30"
Option "Rotate" "none"
Property 'Wacom Wheel Buttons' does not exist on device.
Property 'Wacom Wheel Buttons' does not exist on device.
Property 'Wacom Wheel Buttons' does not exist on device.
Property 'Wacom Wheel Buttons' does not exist on device.
Property 'Wacom Wheel Buttons' does not exist on device.
Property 'Wacom Wheel Buttons' does not exist on device.
Property 'Wacom Strip Buttons' does not exist on device.
Property 'Wacom Strip Buttons' does not exist on device.
Property 'Wacom Strip Buttons' does not exist on device.
Property 'Wacom Strip Buttons' does not exist on device.
Option "Threshold" "26"
Option "Serial" "0"
Option "PressureRecalibration" "on"
Option "PanScrollThreshold" "1300"
```

Unfortunaltely, I'm not familiar with the configuration options. Also, in Xfce I don't find a GUI to configure the buttons. I see that there are some warnings in the output above concerning the buttons, but cannot interpret them. 

I also went through the buttons. The first three of them seem to be mapped:

```
[photon@ThinkpadL13Yoga ~]$ xsetwacom get "Wacom Pen and multitouch sensor Pen stylus" Button 1
button +1 
[photon@ThinkpadL13Yoga ~]$ xsetwacom get "Wacom Pen and multitouch sensor Pen stylus" Button 2
button +2 
[photon@ThinkpadL13Yoga ~]$ xsetwacom get "Wacom Pen and multitouch sensor Pen stylus" Button 3
button +3 
[photon@ThinkpadL13Yoga ~]$ xsetwacom get "Wacom Pen and multitouch sensor Pen stylus" Button 4
4
```

However, only the upper button actually works (emulates the middle mouse button). The lower button, which should emulate the right mouse button has no effect.

Is it possibly related to the option

`Option "Suppress" "2"`

in the output for `xsetwacom get "Wacom Pen and multitouch sensor Pen stylus" all`?

If so, where is it set? I cannot find it in any of the files in /etc/X11 and its subfolders...

Thanks for any help!

**Comment by Photon89** [comment]
Update:

I think, I got a bit further. I ran

`xinput test 16`

(the stylus device) and there, as expected, only the upper button has been registered.

But then I ran 

`xinput test 17`

(the eraser device) and there I only got motion output when the lower button was pressed. So it looks like the lower button switches between the stylus and the eraser device. I configured Xournal++ accordingly and can now use both buttons there.

However, I'd like to emulate the right mouse button with the lower button when outside of Xournal++. What are my options?
